parameter/ElementHandle.cpp

```diff
@@ -124,13 +125,12 @@ bool ElementHandle::getMappingData(const string& strKey, string& strValue) const
 
 bool ElementHandle::getStructureAsXML(std::string &xmlSettings, std::string &error) const
 {
-    string result;
-    if (not mParameterMgr.exportElementToXMLString(&mElement, mElement.getXmlElementName(), result)) {
-        error = result;
-        return false;
-    }
-    xmlSettings = result;
-    return true;
+    // Use default access context for structure export
+    CParameterAccessContext accessContext(error);
+    return mParameterMgr.exportElementToXMLString(
+                &mElement, mElement.getXmlElementName(),
+                CXmlParameterSerializingContext{accessContext, error},
```

Comment:
Coding style inconsistencies: you define `accessContext` a) in its own line and b) using parentheses but you define the serializing context a) in the argument list and b) using braces.